plugins/inputs/vsphere/README.md

```diff
@@ -537,7 +537,7 @@ override the default query interval in the vSphere plugin.
   * CPU: demand, usage, readiness, cost, mhz
   * Datastore: latency, # reads/writes
   * Disk: commands, latency, # reads/writes, provisioned, usage
-  * Memory: granted, usage, active, swap, vmmemctl
+  * Memory: granted, usage, active, swap, vmmemctl, memorySizeMB
```

Comment:
Is this _allocated_ or _provisioned_ memory?

Reply:
Allocated I believe, the library documents it as `Memory size of the virtual machine, in megabytes.`

The library also includes a `MemoryReservation` field, it may be useful to include this as well, what do you think?

Reply:
Yeah would be good to have this as well.